src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceController.kt

```diff
@@ -566,9 +564,8 @@ class LicenceController(private val licenceService: LicenceService) {
     ]
   )
   fun submitLicence(
-    @PathVariable("licenceId") licenceId: Long,
-    @Valid @RequestBody request: SubmitLicenceRequest
+    @PathVariable("licenceId") licenceId: Long
   ) {
-    return licenceService.submitLicence(licenceId, request)
+    return licenceService.submitLicence(licenceId)
   }
```

Comment:
I wonder about this bit ... it is possible for somebody other than the responsible COM to submit a licence. i.e. Someone in their teams. Without accepting the details of the submitter, how do we know this was done by someone else?

Reply:
The username is known from the Auth context!